<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maditor::View::ServerWindow</class>
 <widget class="QWidget" name="Maditor::View::ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6">
    <widget class="Maditor::View::LogsTabWidget" name="logsWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="6">
    <widget class="QTableView" name="modulesWidget"/>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="killButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Kill</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="shutdownButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Shutdown</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="startNoDebugButton">
     <property name="text">
      <string>Start (No Debug)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Maditor::View::LogsTabWidget</class>
   <extends>QTabWidget</extends>
   <header>Logs/logstabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
